AnatomyOfMatplotlib-Part3-HowToSpeakMPL.ipynb

```diff
@@ -9,8 +9,8 @@
     "# Let printing work the same in Python 2 and 3\n",
     "from __future__ import print_function\n",
     "# Turning on notebook plots -- just for use in jupyter notebooks.\n",
+    "%matplotlib inline\n",
```

Comment:
Please don't put the inline backend into these notebooks.

Reply:
I didn't mean to, sorry. Forgot to remove it previously when testing.